src/win/core.c

```diff
@@ -603,3 +603,12 @@ int uv__socket_sockopt(uv_handle_t* handle, int optname, int* value) {
 
   return 0;
 }
+
+
+int uv_cpumask_size(void) {
+#if defined(_WIN64)
+  return 64;
+#else
+  return 32;
+#endif
```

Comment:
The implementation in `master` by #1527 is:

```c
  return (int)(sizeof(DWORD_PTR) * 8);
```

This uses the real type of the system APIs' mask arguments.